### PR TITLE
added the enhancement as stated in #233 Hitting Space Bar Makes The Player pause/play

### PR DESCRIPTION
**Pull request by jeanPokou**
Provide a general summary of your changes in the **Title** above.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [ ] Why is this change required? What problem does it solve?
- [ ] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [ ] Describe your changes in detail.

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [ ] All tests passed.
…Screen Scroll

### CODE REVIEW

File: js/everything.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ function handleYouTubeError(details) {
 // One day, try to move all globals under the ZenPlayer object
 var ZenPlayer = {
     updated: false,
+    isPlaying:false,
```
Comment (shakeelmohamed) [inline js/everything.js]: add a space after `:`

File: js/everything.js [hunk 2]
```diff
@@ -240,6 +241,16 @@ var ZenPlayer = {
                 $("#zen-video-title").attr("href", updatedUrl);
             });
 
+               // event handler for playing : set isPlaying to true
```
Comment (shakeelmohamed) [inline js/everything.js]: fix comment alignment

File: js/everything.js [hunk 3]
```diff
@@ -630,6 +641,23 @@ $(function() {
...
+            if (ZenPlayer.isPlaying) {
+                ZenPlayer.pause();
+            }
+            else {
+                ZenPlayer.play();
+            }
+        }
+    });
+
+        // event handler for keydown , prevent default scrolling handler
```
Comment (shakeelmohamed) [inline js/everything.js]: fix comment alignment, remove the space before `,`

File: js/everything.js [hunk 4]
```diff
@@ -630,6 +641,23 @@ $(function() {
...
+    $(document).on("keydown", function(evt) {
+        evt.preventDefault();
```
Comment (shakeelmohamed) [inline js/everything.js]: Shouldn't we only do this for `evt.keyCode === 32`?

File: js/everything.js [hunk 5]
```diff
@@ -630,6 +641,23 @@ $(function() {
     });
     // Load the player
     ZenPlayer.init(currentVideoID);
+
+    $(document).on("keyup", function(evt) {
+        evt.preventDefault();
+        if (evt.keyCode === 32) {
```
Comment (shakeelmohamed) [inline js/everything.js]: Add a comment here saying which key this is

File: package.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
     "lint:js": "eslint js/*.js",
     "lint:tests": "eslint test/*.js",
     "lint:html": "htmlhint index.html",
-    "lint:css": "csslint --format=compact --warnings=box-model,ids css/*.css",
+    "lint:css": "csslint --format=compact --warnings=box-model,ids css/",
```
Comment (shakeelmohamed) [inline package.json]: Why this change?

Comment (jeanPokou) [inline package.json]: I was getting "file not found"  with csslint using the global pattern  "*.css", having this way make csslint check every file in the folder not only css, you can revert it, although it very obvious that a css folder should contain css files

File: js/everything.js (hunk 5 above)
Comment (jeanPokou) [inline js/everything.js]: ASCII code 32 === "space bar" , you can comment the code

File: package.json (hunk 6 above)
Comment (shakeelmohamed) [inline package.json]: Sure, it just seems odd that it hasn't been an issue until now.

File: js/everything.js (hunk 5 above)
Comment (shakeelmohamed) [inline js/everything.js]: Please add the comment

File: js/everything.js (hunk 4 above)
Comment (shakeelmohamed) [inline js/everything.js]: @jeanPokou without the conditional, this is causing many issues (arrow keys, tab, all alphanumeric, etc. keys are being swallowed).

We also want to make sure we don't play/pause while the user is typing in the the search box

File: js/everything.js [hunk 7]
```diff
@@ -630,6 +641,23 @@ $(function() {
     });
     // Load the player
     ZenPlayer.init(currentVideoID);
+
+    $(document).on("keyup", function(evt) {
+        evt.preventDefault();
```
Comment (shakeelmohamed) [inline js/everything.js]: Se my comment for the `keydown` event handler. We need to make sure we don't play/pause while the user is typing in the search box

**Comment by shakeelmohamed** [comment]
@jeanPokou Thanks for getting started on this change, but there's a bit of work remaining. See my inlined comments

**Comment by shakeelmohamed** [comment]
Usurped by #242, thanks for your work here @jeanPokou. Welcome to the org!
